torchinductor/lowering.py

```diff
@@ -351,13 +351,18 @@ def broadcast_tensors(*inputs):
     return outputs
 
 
-@register_lowering(
-    [aten.alias, aten.detach, aten.lift_fresh_copy, aten.lift_fresh, aten.lift]
-)
+@register_lowering([aten.alias, aten.detach, aten.lift_fresh, aten.lift])
 def detach(x):
     return x  # AOT autograd handles this for us
 
 
+if hasattr(aten, "lift_fresh_copy"):
```

Comment:
I think this might need to do a copy. Although I suppose it depends on how inductor lowers something like this.

```
def forward(x):
    val = self._tensor_constant0
    val.add_(1)
    return val
```